slapp/transforms/image_utils.py

```diff
@@ -0,0 +1,71 @@
+from typing import Tuple, Union
+from pathlib import Path
+from math import ceil
+
+import numpy as np
+from PIL import Image, ImageDraw, ImageFont
+
+
+def add_scale(image: Union[Path],
+              scale_position: Tuple[int, int],
+              font_file: Path,
+              resolution: float = 10 / 13,
```

Comment:
this is not exact. If you're going to specify a float, why not make it exact. `400 / 512` or `25 / 32` or `0.78125`

Reply:
Good call